tests/README.md

```diff
@@ -9,17 +9,22 @@
 ### All tests
 
 ```bash
-go test -v -tags e2e setup_test.go        # Only needs to be run once.
-go test -v -tags e2e ./scalers_go/...
-go test -v -tags e2e cleanup_test.go      # Skip if you want to keep testing.
+go test -v -tags e2e ./tests/utils/setup_test.go        # Only needs to be run once.
+go test -v -tags e2e ./tests/scalers_go/...
+go test -v -tags e2e ./tests/utils/cleanup_test.go      # Skip if you want to keep testing.
```

Comment:
Do you think that we need to specify `tests` folder? this readme is already inside `tests` folder, so being strict, this path doesn't work from the place where the readme is

Reply:
good point, let me fix that

Reply:
fixed